src/app/shared/models/classes/coffee.class.ts

```diff
@@ -4,9 +4,9 @@ import { Product } from "./product.class";
 export class Coffee extends Product {
     productType: ProductType = "coffee";
     origin: string | null = null;
-    tastingNotes: string | null = null;
+    tastingNotes: string[] | null = null;
     process: string | null = null;
-    varietal: string | null = null;
+    varietal: string[] | null = null;
```

Comment:
I included varietal with this change since it seems strictly related in terms of content/changes (though not in terms of the  branch title).